spec/index.html

```diff
@@ -172,7 +172,7 @@ <h3>Simple Statements</h3>
       or <a data-cite="RDF12-CONCEPTS#dfn-iri">IRI</a>) labeling what
       <a data-cite="RDF12-CONCEPTS#dfn-named-graph">named graph</a>
       in a <a data-cite="RDF12-CONCEPTS#dfn-rdf-dataset">dataset</a> the triple belongs to.
-      White space (spaces <code>U+0020</code> or tabs <code>U+0009</code>) may surround terms,
+      White space (spaces, code point <code class="codepoint">U+0020</code>, or tabs, code point <code class="codepoint">U+0009</code>) may surround terms,
```

Comment:
```suggestion
      White space (spaces, code point <code class="codepoint">U+0020</code>, and/or tabs, code point <code class="codepoint">U+0009</code>) may surround terms,
```

Reply:
Is that “may” intended to be an RFC2119 “MAY”? If not, we should use another word.

Reply:
`is permitted to`? 
`can`?
I don't think I put `may` into this sentence.